core/src/test/java/org/elasticsearch/action/support/IndicesOptionsTests.java

```diff
@@ -60,29 +60,82 @@ public void testSerialization() throws Exception {
     }
 
     public void testFromOptions() {
-        int iterations = randomIntBetween(5, 20);
-        for (int i = 0; i < iterations; i++) {
```

Comment:
yes I removed it intentionally.I do not think it's that important to test all combinations in one run here. I can restore if you'd like.